fineract-provider/build.gradle

```diff
@@ -164,15 +171,15 @@ configurations.driver.each {File file ->
 task createDB {
     description= "Creates the MariaDB Database. Needs database name to be passed (like: -PdbName=someDBname)"
     doLast {
-        def sql = Sql.newInstance( 'jdbc:mariadb://localhost:3306/', mysqlUser, mysqlPassword, 'org.mariadb.jdbc.Driver' )
+        def sql = Sql.newInstance( 'jdbc:mariadb://localhost:3307/', mysqlUser, mysqlPassword, 'org.mariadb.jdbc.Driver' )
```

Comment:
Please reverse these changes!

Reply:
will do that

Reply:
This comment shouldn't be resolved since the changes weren't yet made.